features/storage/TESTS/blockdevice/general_block_device/main.cpp

```diff
@@ -47,13 +48,27 @@
 #include "FlashIAPBlockDevice.h"
 #endif
 
+// Debug available
+#ifndef MODE_DEBUG
```

Comment:
I thought we had other debug macros.
Or is this the pattern used within the storage tests?

Reply:
@theamirocohen @davidsaada Would appreciate an answer here (I realise I only asked four hrs ago), but won't hold up the merge on it.

If it need to be fixed, it can be brought in later.

Reply:
This is unrelated to other debug macros. Original test code poured a lot of unnecessary prints during the test. Current change removed them (among the rest), by simply using the `DEBUG_PRINT` define, which is turned off by default depending on `MODE_DEBUG`. Guess these prints could have been removed altogether, but @theamirocohen took precautions by keeping them in for the case someone should want to see them while debugging.